src/components/CreateBook.js

```diff
@@ -28,28 +28,38 @@ const CreateBook = () => {
 
   return (
     <div>
-      <h2>Add New Book</h2>
-      <div className="form">
-        <input
-          className="input"
-          type="text"
-          placeholder="Book title"
-          value={book.title}
-          name="title"
-          onChange={handleChange}
-        />
-        <input
-          type="text"
-          placeholder="Author name"
-          value={book.author}
-          name="author"
-          onChange={handleChange}
-        />
-
-        <button className="button" onClick={onSaveBookClicked} type="button">
-          Submit
-        </button>
+      <div className="bookform">
+        <h1 className="heading">Create Book</h1>
```

Comment:
You've made good progress by adding a title to your project. However, to align with the design, it's important to update the styling. Please make the following adjustments:

1. **Styling**: Ensure that the styling of your title closely matches the design specifications, including font size, font weight, color, and other relevant attributes.

2. **Border at the Top**: Add a border at the top of the title, as specified in the design.

3. **Margin and Padding**: Adjust the margin and padding of the title to match the design's spacing requirements.

By making these changes, you'll bring your title in line with the design, creating a more visually consistent and polished user interface. Keep up the great work! 🌟🎨